services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:

```

Comment:
You must show inner type of `QuerySet[CinemaHall]` in type hint. Fix everywhere, please